docs/rdd-programming-guide.md

```diff
@@ -442,7 +442,7 @@ Apart from text files, Spark's Python API also supports several other data forma
 
 **Writable Support**
 
-PySpark SequenceFile support loads an RDD of key-value pairs within Java, converts Writables to base Java types, and pickles the
+PySpark SequenceFile support loads an RDD of key-value pairs within Java, converts [Writables](https://hadoop.apache.org/docs/current/api/org/apache/hadoop/io/Writable.html) to base Java types, and pickles the
```

Comment:
nit: Maybe `Writables` -> `Writable` ??
Seems like Writable is an official naming.

Reply:
Yeah I might write `[Writable](..)s` but that's awkward. `[Writable](...) objects`?

Reply:
Sounds good!

Reply:
I will rather spend the time on another case of improving readability/understandability of the docs. I don't see these nits as something that is towards the mentioned goals.